src/framework/preferences/js/LetterSpaceEnactor.js

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2017 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+var fluid_3_0_0 = fluid_3_0_0 || {};
+
+(function ($, fluid) {
+    "use strict";
+
+    /*******************************************************************************
+     * letterSpace
+     *
+     * Sets the letter space on the container to the number of units to increase
+     * the letter space by. If a negative number is provided, the space between
+     * characters will decrease. Setting the size to 0 will use the default letter
```

Comment:
It seems using the default letter space is when the size (`value`) is set to 1, instead of 0.